src/tests/test_claim_types.py

```
@@ -0,0 +1,296 @@
+    
+    return claim, evidence
+
+
+@pytest.mark.asyncio
+@patch("src.verifact_manager.Runner.run")
+async def test_true_claim(mock_run, manager):
+    """Test the pipeline with a true claim."""
+    # Get a true verdict from fixtures
+    true_verdict = get_verdict_by_type("true")
+    claim, evidence = get_claim_and_evidence_for_verdict(true_verdict)
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)